packages/spec/vaults/vaults.spec.ts

```diff
@@ -341,94 +440,214 @@ describe('Given the Aave Vaults', () => {
     });
 
     describe(`When the organization changes the vault's fee`, () => {
-      const organization = createNewWallet();
-
-      it('Then the new fee should be reflected in the vault object', async ({
-        annotate,
-      }) => {
-        annotate(`organization address: ${organization.account!.address}`);
-        const initialVault = await createVault(organization);
-        assertOk(initialVault);
+      describe("And is single recipient of the vault's fees", () => {
+        const organization = createNewWallet();
+
+        it('Then the new fee should be reflected in the vault object', async ({
+          annotate,
+        }) => {
+          annotate(`organization address: ${organization.account!.address}`);
+          const initialVault = await createVault(organization);
+          assertOk(initialVault);
+
+          const newFee = bigDecimal('4.60');
+          const updateResult = await vaultSetFee(client, {
+            chainId: initialVault.value.chainId,
+            vault: initialVault.value.address,
+            newFee: newFee,
+          })
+            .andThen(sendWith(organization))
+            .andTee((tx) => annotate(`tx to set fee: ${tx.txHash}`))
+            .andThen(client.waitForTransaction);
+          assertOk(updateResult);
+
+          const newVaultInfo = await vault(client, {
+            by: { address: initialVault.value.address },
+            chainId: initialVault.value.chainId,
+          });
+          assertOk(newVaultInfo);
```

Comment:
We could still use the `.andThen(...`


```suggestion
            .andThen(client.waitForTransaction)
            .andThen(() => vault(client, {
              by: { address: initialVault.value.address },
              chainId: initialVault.value.chainId,
          });
          assertOk(updateResult);  
```